secure.go

```diff
@@ -124,6 +127,28 @@ func (s *Secure) Handler(h http.Handler) http.Handler {
 			return
 		}
 
+		// No headers will be written to the ResponseWriter.
```

Comment:
Is this correct? Doesn't this method write the headers?

Reply:
Nice catch :)